src/gatling/scala/SingleLeaseMultipleChecks.scala

```diff
@@ -0,0 +1,45 @@
+import com.typesafe.config.{Config, ConfigFactory}
+import com.warrenstrange.googleauth.GoogleAuthenticator
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+import scala.concurrent.duration._
+
+class SingleLeaseMultipleChecks extends Simulation {
+  val config: Config = ConfigFactory.load()
+
+  val authenticator: GoogleAuthenticator = new GoogleAuthenticator()
+
+  val requestJwt = exec(
+    http("Lease token")
+      .post("/lease")
+      .formParam("microservice", config.getString("service.name"))
+      .formParam("oneTimePassword", "${otp}")
+      .check(bodyString.saveAs("jwt"))
+  )
+
+  val checkJwt = exec(
+    http("Check token")
+      .get("/details")
+      .header("Authorization", "Bearer ${jwt}")
+      .check(substring(config.getString("service.name")))
+  )
+
+  val otpFeeder = Iterator.continually(Map("otp" -> authenticator.getTotpPassword(config.getString("service.pass"))))
+
+  setUp(
+    scenario("Testing")
+      .feed(otpFeeder)
+      .exec(requestJwt)
+      .repeat(config.getInt("repeats")) {
+        exec(checkJwt)
+      }
+      .inject(
+        rampUsers(10000) over (1 minute)
```

Comment:
rampUsers is fine but the number is too great. We can be pretty sure s2s won't handle that many (probably Gatling won't be able to generate such load, either). Besides, we won't have 10000 instances of services hitting s2s. 50? 100?